src/drive/web/modules/actionmenu/MobileActionMenu.jsx

```diff
@@ -65,14 +65,19 @@ const MenuHeaderFile = ({ file, t }) => {
               {extension}
             </span>
           </div>
-          <div className="u-coolGrey u-fz-tiny u-fs-normal u-flex u-flex-items-center">
-            <Icon icon="qualify" size="10" />
-            <Caption
-              className={classNames(styles['fil-mobileactionmenu-category'])}
-            >
-              {t(`Scan.items.${file.metadata.label}`)}
-            </Caption>
-          </div>
+          {file.metadata &&
+            file.metadata.label && (
```

Comment:
I find `has(file, 'metadata.label')` more elegant and less error-prone, but not blocking for me.

Reply:
I'm waiting for file?.metadata.?label :D